src/parser/core/modules/Data.ts

```diff
@@ -25,6 +27,11 @@ export class Data extends Module {
 		return getDataBy(this.statuses, 'id', id)
 	}
 
+	getActionAppliedByStatus(status: Status) {
```

Comment:
this seems like a very specialised use-case to be putting in the low-level data module...

is it feasible to keep in a more contained module scope? (also the fn name seems the wrong way around, almost - statuses don't apply actions)

Reply:
Whoops, I must have been drunk when I named that.  

I added the helper method for testability.  I spent about 30 minutes hacking at trying to mock statusesApplied the way we currently do it and had only heartbreak and sorrow, so I tried the (mockable) helper method instead.